NTC-Weather-Forecast-COVID-19/NTC-Weather-Forecast-COVID-19/Controller/Weather Forecast/SettingTempViewController.swift

```diff
@@ -8,8 +8,57 @@
 import UIKit
 
 final class SettingTempViewController: UIViewController {
+    @IBOutlet private weak var settingTempTableView: UITableView!
 
+    private var currentIndexTempFormat = 0
+    private let identifierCell = "SettingTempTableViewCell"
+    private let keyTempFormat = UserDefaultsKeys.keyTempFomat.rawValue
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        settingTempTableView.delegate = self
+        settingTempTableView.dataSource = self
+        loadTempFormatFromUserDefaults()
+    }
+    
+    private func updateTempFormatInUserDefaults(newValue: Int) {
+        UserDefaults.standard.set(newValue, forKey: keyTempFormat)
+    }
+    
+    private func loadTempFormatFromUserDefaults() {
+        if let indexTempFormat = UserDefaults.standard.value(forKey: keyTempFormat) as? Int {
+            currentIndexTempFormat = indexTempFormat
+        } else {
+            currentIndexTempFormat = 0
+        }
+    }
+}
+
+extension SettingTempViewController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return TempFormat.allCases.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: identifierCell, for: indexPath)
+        let cellText = TempFormat.allCases[indexPath.row].rawValue.components(separatedBy: " ")
+        cell.textLabel?.text = cellText.first
+        cell.detailTextLabel?.text = cellText[1]
+        if indexPath.row == currentIndexTempFormat {
+            cell.accessoryType = .checkmark
+        }
+        return cell
+    }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.visibleCells.forEach { (cell) in
+            cell.accessoryType = .none
+        }
```

Comment:
xử lý gì ở đây thế thanh niên?
Sao lại chạy for ở đây

Reply:
vâng. Đoạn đó xử lý k cần thiết nên e bỏ r ạ.